game-core/src/main/java/games/strategy/triplea/TripleAPlayer.java

```diff
@@ -566,7 +566,6 @@ private void endTurn() {
     if (getPlayerBridge().isGameOver()) {
       return;
     }
-    final GameData data = getGameData();
```

Comment:
It's a bit amusing this is used almost 20 lines later.   : )

"Clean code" was written in 2008, this code almost certainly predates it. Interesting to see the concept of 'good code' evolve.